doc/Developing.md

```diff
@@ -47,7 +47,8 @@ For Windows instructions [click here](#building-on-windows).
  - [**Troubleshooting**](#troubleshooting)
      - [Linux](#linux)
      - [Windows](#windows)
-     - [macOS](#macos)
+     - [macOS](#macOS)
+     - [RHEL7](#RHEL7)
```

Comment:
Some of the people reporting similar errors online were using CentOS, so I might go with RHEL7 / CentOS7 here.